baggers.js

```diff
@@ -16416,91 +16416,125 @@ var data = {
       },
     },
     {
-      since: '2015-03-10',
```

Comment:
I don't understand why this is removed. Could you check?
Make sure you put also your addition at the end of the file.